homeassistant/components/fronius/__init__.py

```diff
@@ -42,26 +42,27 @@
 
 _FroniusCoordinatorT = TypeVar("_FroniusCoordinatorT", bound=FroniusCoordinatorBase)
 
+FroniusConfigEntry = ConfigEntry["FroniusSolarNet"]
```

Comment:
```suggestion
FroniusConfigEntry = ConfigEntry[FroniusSolarNet]
```

Reply:
The quotes are necessary. `FroniusSolarNet` is only defined below.

Reply:
> undefined Name

`FroniusSolarNet` is defined further down in `__init__.py`